test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml

```diff
@@ -80,7 +89,37 @@ spec:
           mountPath: /dev
         - name: root-mount
           mountPath: /root
+        - name: cos-tools
+          mountPath: /build/cos-tools
+        command:
+        - bash
+        - -c
+        - | 
+          echo "Checking for existing GPU driver modules"
+          if lsmod | grep nvidia; then
+            echo "GPU driver is already installed, the installed version may or may not be the driver version being tried to install, skipping installation"
+            exit 0
+          else
+            echo "No GPU driver module detected, installing now"
+            /cos-gpu-installer install
+          fi
+      - image: "gcr.io/gke-release/nvidia-partition-gpu@sha256:e226275da6c45816959fe43cde907ee9a85c6a2aa8a429418a4cadef8ecdb86a"
+        name: partition-gpus
+        env:
+          - name: LD_LIBRARY_PATH
+            value: /usr/local/nvidia/lib64
+        resources:
+          requests:
+            cpu: 150m
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: nvidia-install-dir-host
+          mountPath: /usr/local/nvidia
+        - name: dev
+          mountPath: /dev
+        - name: nvidia-config
+          mountPath: /etc/nvidia
       containers:
-      - image: "registry.k8s.io/pause:3.9"
+      - image: "gcr.io/google-containers/pause:2.0"
```

Comment:
we should revert this line back, also the upstream source should switch to registry.k8s.io, `grc.io/google-containers` is a) really old b) not actively supported c) GCR is going away next year.

Reply:
Yes, "registry.k8s.io/pause:3.9" is better.

Reply:
+1 on revert back